### PR TITLE
Selectors and cascading bugs

### PR DESCRIPTION
**Pull request by ivannemeth**

### CODE REVIEW

File: css-structure/css-structure_selectors-and-cascading/css/styles.css [hunk 1]
```diff
@@ -49,92 +49,93 @@ nav li {
...
-FIX_ME {
+a[target="_blank"] {
   outline: 2px dashed red;
   color: #ff8000;
 }
 
 /* 4. These styles should apply to all links in the main navigation */
-FIX_ME {
+a[href="#"] {
   outline: 2px dashed red;
```
Comment (thisKonrad) [inline css-structure/css-structure_selectors-and-cascading/css/styles.css:73]: main nav a

File: css-structure/css-structure_selectors-and-cascading/css/styles.css [hunk 2]
```diff
@@ -49,92 +49,93 @@ nav li {
...
-FIX_ME {
+.list .list {
   outline: 2px dashed red;
   margin: 0.5rem 0 0 0;
   background-color: #f1f1f1;
 }
 
 /* 14. These styles should apply to all list items in nested lists */
-FIX_ME {
+ul > li > ul > li {
```
Comment (thisKonrad) [inline css-structure/css-structure_selectors-and-cascading/css/styles.css:138]: i did only: ul ul li

File: css-structure/css-structure_selectors-and-cascading/css/styles.css (hunk 1 above)
Comment (ivannemeth) [reply css-structure/css-structure_selectors-and-cascading/css/styles.css:73]: haste recht, sieht besser aus

File: css-structure/css-structure_selectors-and-cascading/css/styles.css (hunk 2 above)
Comment (ivannemeth) [reply css-structure/css-structure_selectors-and-cascading/css/styles.css:138]: both are ok, i tested it

**Pull request merged**